scripts/custom-prereq/kafka/kafka.spec

```diff
@@ -12,7 +12,6 @@ URL:            https://kafka.apache.org/
 Source0:	kafka_%{_kafka_version}-%{_kafka_release}.tgz
 Source1:	kafka.service
 Source2:	kafka-zookeeper.service
-Requires:	java-1.8.0-openjdk-headless
```

Comment:
Have we tested Kafka package generation and installation with this. If not we can skip this change

Reply:
Not tested. Skipping it.

Reply:
Done